api/src/endpoints/svgThumbnail.js

```diff
@@ -7,10 +7,25 @@ const VALID_MODELS = models.map((m) => m.short_name);
 
 const routes = express.Router();
 
+function getDimension(dimensionString) {
+  const minDim = 100;
+  const maxDim = 600;
+  if(dimensionString){
+    if (!/^\d+$/.test(dimensionString)) {
+      throw new Error(`Invalid dimension provided: ${dimensionString}. The width and/or height parameters should be an integer between ${minDim} and ${maxDim}.`);
+    } else {
+      var myDimension = parseInt(dimensionString) || null;
+      if ((myDimension < minDim || myDimension > maxDim)){
+        throw new Error(`Invalid dimension provided: ${dimensionString}. The width and/or height parameters should be an integer between ${minDim} and ${maxDim}.`);
```

Comment:
This error message is repeated twice. Would we like to create a file similar to `frontend/src/content/messages.js` and define it there?

Reply:
I'm not sure how the variable inclusion would work - those strings are static.

Reply:
We could define a function that takes  `dimensionString`,  `minDim`, and `maxDim` as arguments, either here or in a `messages.js` similar file. But I'm fine with keeping it as is as well

Reply:
It's a good point, but I think we should have a separate issue for this. `messages.js` in under `/frontend`, whereas this is under `/api`, and once we have a way of dealing with arguments in strings then I guess a lot more strings could be migrated to `messages.js`.
I'm leaving this conversation open for others to join. Also, if there is a new issue for this, it would be good to have it linked to this conversation.

Reply:
Just to clarify, the suggestion was not to use file `frontend/src/content/messages.js`, but to create a file with similar purpose in the api folder

Reply:
I think it's a good idea to have a new issue for backend error messages. For now I do think it would be nice to refactor the function and use a variable for the two instances of the same error message.

Reply:
ebc272d combines the 2 error messages into 1.

Reply:
Nice commit! Created issue #755 for having a common file for error messages